createdb.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 
 ## These two lines are needed to run on EL6
-__requires__ = ['SQLAlchemy >= 0.7', 'jinja2 >= 2.4']
+__requires__ = ['SQLAlchemy >= 0.8', 'jinja2 >= 2.4']
```

Comment:
On Wed, Jan 21, 2015 at 07:45:21AM -0800, Ralph Bean wrote:
>    In createdb.py:
> 
>  > @@ -1,7 +1,7 @@
>  >  #!/usr/bin/python
>  > 
>  >  ## These two lines are needed to run on EL6
>  > -__requires__ = ['SQLAlchemy >= 0.7', 'jinja2 >= 2.4']
>  > +__requires__ = ['SQLAlchemy >= 0.8', 'jinja2 >= 2.4']
> 
>    do we need to worry about el6 anymore? Can these lines be removed?

We do not have to since we are running EL7 but I do not know if rpmfusion is
running 6 or 7 and they expressed interest in pkgdb. So for the moment I prefer
to remain EL6 compatible :)